src/Utils/TypeInfo.php

```diff
@@ -437,57 +427,37 @@ private static function getFieldDefinition(Schema $schema, Type $parentType, Fie
     /**
      * @param NamedTypeNode|ListTypeNode|NonNullTypeNode $inputTypeNode
      *
-     * @return Type|null
-     *
      * @throws InvariantViolation
      */
-    public static function typeFromAST(Schema $schema, $inputTypeNode)
+    public static function typeFromAST(Schema $schema, $inputTypeNode) : ?Type
     {
         return AST::typeFromAST($schema, $inputTypeNode);
     }
 
-    /**
-     * @return Directive|null
-     */
-    public function getDirective()
+    public function getDirective() : ?Directive
     {
         return $this->directive;
     }
 
-    /**
-     * @return FieldDefinition
-     */
-    public function getFieldDef()
+    public function getFieldDef() : ?FieldDefinition
     {
-        if (! empty($this->fieldDefStack)) {
-            return $this->fieldDefStack[count($this->fieldDefStack) - 1];
-        }
-
-        return null;
+        return $this->fieldDefStack[count($this->fieldDefStack) - 1] ?? null;
     }
 
     /**
      * @return mixed|null
      */
     public function getDefaultValue()
     {
-        if (! empty($this->defaultValueStack)) {
-            return $this->defaultValueStack[count($this->defaultValueStack) - 1];
-        }
-
-        return null;
+        return $this->defaultValueStack[count($this->defaultValueStack) - 1] ?? null;
     }
 
     /**
-     * @return ScalarType|EnumType|InputObjectType|ListOfType|NonNull|null
+     * @return (Type & InputType) | null
```

Comment:
you don't have spaces elsewhere

```suggestion
     * @return (InputType&Type)|null
```